scripts/DocParser.js

```diff
@@ -124,6 +125,47 @@ function validate (docs, name, componentDirectory, strict) {
 	}
 }
 
+function parseTableOfContents (frontMatter, body) {
+	let maxdepth;
```

Comment:
Would it make sense to set the default depth to `2` and only adjust when needed?  The default depth of `markdown-toc` seems pretty deep.

Reply:
For our purposes, I'm not sure it matters very much. I agree it's deep but it might be surprising for levels to be hidden when the author expected them to be visible. However, if you think we should limit it to 2 levels as the standard and anything else should be handled as a deviation, I can support that.

Reply:
I don't feel super strongly about it.  If `2` seems like a comfortable level, let's use it.  Particularly if we float this right in the future, we probably don't want it to be very deep.